pkg/engine/engine.go

```diff
@@ -233,8 +234,14 @@ func parseRefToArgs(c ContainerRef) ([]string, error) {
 		args = append(args, "--rm")
 	}
 
+	// If PublishAll is set - publish all of the ports. Otherwise, bind each
+	// one individually to localhost.
 	if c.PublishAll {
```

Comment:
There are no more references that can reach `PublishAll` anymore, are there?  Should this just be removed?

Reply:
I had some time yesterday so I added a hidden flag that emits a warning log if it's used.